src/components/common/Extensions/UserHub/UserHub.tsx

```diff
@@ -120,7 +124,10 @@ const UserHub: FC<Props> = ({ initialOpenTab = UserHubTab.Balance }) => {
                           'font-medium': selectedTab === id,
                         })}
                       >
-                        <span className="mr-2 flex shrink-0">
+                        <span className="relative mr-2 flex shrink-0">
+                          {id === UserHubTab.Notifications && (
+                            <CountBadge count={unreadCount} maximum={99} />
+                          )}
```

Comment:
This might just be a matter of preference, but I prefer to keep the UI and logic separate. In my opinion, it would be better to move this part of the code:
```
if (!count || count < 1) {
    return null;
  }
```
into this component, so that CountBadge always renders something.

it would look like this:
```suggestion
                          {id === UserHubTab.Notifications && (!unreadCount || unreadCount < 1) && (
                            <CountBadge count={unreadCount} maximum={99} />
                          )}
```

Reply:
I think I'd prefer to keep it inside the component as I think `!unreadCount || unreadCount < 1` (or an equivalent) would end up being used to wrap every use of this component.

This pattern also exists in a bunch of other components which return null at the top of the component based on a conditional check.